cli/flox/doc/flox-activate.md

```diff
@@ -29,7 +29,14 @@ interactive shell.
 Launches a subshell non-interactively when invoked with a command
 and arguments.
 May also be invoked as `$(flox activate)` to produce commands to be sourced
-by your current `$SHELL`.
+by the parent shell.
+The parent shell is detected automatically by inspecting the executable path
+of the parent process.
+If Flox can not interpret the path as a shell or a subshell is launched,
+Flox will fall back to `$SHELL`.
```

Comment:
nit: it was a little confusing to try to understand both invocation modes described in the same sentence. Might be more clear to split up

Reply:
better this way?